packages/core/src/Stores/common-store.js

```diff
@@ -150,7 +151,7 @@ export default class CommonStore extends BaseStore {
     }
 
     @action.bound
-    routeBackInApp(history) {
+    routeBackInApp(history, additional_platform_path = []) {
```

Comment:
An alternate to this approach would be to use a similar approach to how SmartTrader is registered, by setting the `is_external` flag. The name of the property would need to change to fit P2P, since P2P is not external, however the implementation would be the same.

With that said, having an `additional_platform_path` to allow on-the-go "back" routing in the app seems like a useful thing to have, so I think no changes required for this comment.

Reply:
Thanks @mahdipourziaei-binary 🙇